.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: Build docker container for the SP Dashboard
+
+
+on:
+  push:
+    branches: [ feature/docker ]
+  pull_request:
+    branches: [ feature/docker ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Check out the repo
+      uses: actions/checkout@v2
+
+    - name: Build the php-fpm container and push to GitHub Packages
+      uses: docker/build-push-action@v1
+      with:
+        dockerfile: docker/php-fpm/Dockerfile
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+        registry: docker.pkg.github.com
+        repository: surfnet/sp-dashboard/spdashboard_php-fpm
+        tag_with_ref: true
+
+    - name: Build the Apache container and push to GitHub Packages
+      uses: docker/build-push-action@v1
+      with:
+        dockerfile: docker/httpd/Dockerfile
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+        registry: docker.pkg.github.com
+        repository: surfnet/sp-dashboard/spdashboard_web
+        tag_with_ref: true
```

Comment:
A step could be added to run tests but then it would be better to move the Ant tasks to Composer. This was already done in other projects. This would also be beneficial right now because Java wouldn't be needed for the Travis builds.

Reply:
Let's make it so after merging this branch :)